examples/jsm/loaders/EXRLoader.js

```diff
@@ -2462,24 +2472,100 @@ class EXRLoader extends DataTextureLoader {
 
 			// RGB images will be converted to RGBA format, preventing software emulation in select devices.
 			let fillAlpha = false;
+			let invalidOutput = false;
 
+			// Validate if input texture contain supported channels
 			if ( channels.R && channels.G && channels.B ) {
 
-				fillAlpha = ! channels.A;
 				EXRDecoder.outputChannels = 4;
-				EXRDecoder.decodeChannels = { R: 0, G: 1, B: 2, A: 3 };
 
 			} else if ( channels.Y ) {
 
 				EXRDecoder.outputChannels = 1;
-				EXRDecoder.decodeChannels = { Y: 0 };
 
 			} else {
 
 				throw new Error( 'EXRLoader.parse: file contains unsupported data channels.' );
 
 			}
 
+			// Setup output texture configuration
+			switch ( EXRDecoder.outputChannels ) {
+
+				case 4:
+
+					if ( outputFormat == RGBAFormat ) {
+
+						fillAlpha = ! channels.A;
+						EXRDecoder.format = RGBAFormat;
+						EXRDecoder.colorSpace = LinearSRGBColorSpace;
+						EXRDecoder.outputChannels = 4;
+						EXRDecoder.decodeChannels = { R: 0, G: 1, B: 2, A: 3 };
+
+					} else if ( outputFormat == RGFormat ) {
+
+						EXRDecoder.format = RGFormat;
+						EXRDecoder.colorSpace = NoColorSpace;
```

Comment:
I think the color space should be `LinearSRGBColorSpace` in all these cases -- not `NoColorSpace`.

Actually, the color space should be based on the chromaticity coordinates defined in the file, but so far, three.js has ignored that, and required the user to specify the color space.

Reply:
I only meant to suggest that the color space should be set to `LinearSRGBColorSpace`.

But that can be change in a later PR if others agree to the suggestion.